Examples/Scripts/SetUpSPECIES.sh

```diff
@@ -0,0 +1,471 @@
+#! /bin/bash
+
+# Species specific config file 
+
+# Takuya Hayashi, RIKEN Japan
+# Akiko Uematsu, RIKEN, Japan
+# Yuki Hori, RIKEN Japan
+# Chad Donahue, Washington University St. Louis, USA
+# Matthew F Glasser, Washington University St. Louis, USA
+# Copyright (c) 2016-2023
+# All rights reserved.
+
+## Species specific variables
+if [[ "$SPECIES" == *Human* ]] ; then
```

Comment:
With the change to argument-passing in PreFSBatchNHP, this should either be using `$1`, or do `SPECIES=$1` near the top.  At present, it is still looking at the value assigned to `$SPECIES` in the sourcing code, so a bare `source SetUpSPECIES.sh Macaque 0.35` without setting `$SPECIES` in the batch script would fail.  Similar for `$2` versus `$StrucRes`.

Reply:
${UnwarpDir} needs to be passed in as an argument.

Reply:
Also MION

Reply:
If UnwarpDir isn't used to set filenames, etc, then this file shouldn't need to know about it (and I think it shouldn't be responsible for setting it, just like it doesn't set the input filenames).

Looks like MION is only used to change the fix training data, and currently uses Macaque MION training data regardless of species (I would guess training doesn't exist for other species).  Again, this is specific to the input data, not necessarily global to the species, so I would consider not having the species setup file interact with it.  Instead, the batch file can have a section that says something like:

```
#if you used MION, change the fix training data by uncommenting this line and providing the path to the training file if necessary.
#TrainingData=NHPHCP_Macaque.USPIO
```